.konflux/overlay/overlay.bash

```diff
@@ -268,14 +268,17 @@ overlay_release()
 {
     echo "Overlaying release..."
 
-    local display_name="lifecycle-agent-operator"
-    local description="lifecycle-agent-operator"
+    local display_name="Lifecycle Agent"
+    local description="# Lifecycle Agent for OpenShift\nThe Lifecycle Agent for OpenShift
```

Comment:
the new data is sourced from the migrated 4.18 catalog
https://github.com/openshift-kni/lifecycle-agent/blob/release-4.18/.konflux/catalog/lifecycle-agent/catalog.yaml#L233